torch/csrc/jit/passes/utils/memory_dag.h

```diff
@@ -116,6 +111,7 @@ enum class BfsDirection {
   POINTED_FROM,
 };
 
+std::unordered_set<const Element*> convert(MemoryLocations bits);
```

Comment:
Can we make this name more descriptive and add a comment about it?

Reply:
Resolved by https://github.com/pytorch/pytorch/pull/20899#discussion_r287478737